toolkit/json.py

```diff
@@ -29,8 +29,8 @@
 """
 
 try:
-    from json import *
+    from .json import *
     dumps
 except (ImportError, NameError):
-    import json as simplejson
+    from . import json as simplejson
     #from simplejson import *
```

Comment:
I don't think this file is needed anymore as you can achieve what it does with the builtin json.

Reply:
Done in f0f1d5c